src/Sql/dbo/Stored Procedures/User_Create.sql

```diff
@@ -29,7 +29,8 @@
     @KdfIterations INT,
     @CreationDate DATETIME2(7),
     @RevisionDate DATETIME2(7),
-    @ApiKey VARCHAR(30)
+    @ApiKey VARCHAR(30),
+    @ForcePasswordReset BIT
```

Comment:
@kspearrin , I think when adding parameters like this going forward into the future, it would be good to set default values on them making them optional, then when it comes to time for deployment, if time passes between the DB update and the API deployment/server components, they will not start failing and we can maintain reverse compatibility.

I would propose something like this look similar to:
```suggestion
    @ForcePasswordReset BIT = 0
```

Reply:
Agreed.

Reply:
Thanks guys. Updated.